Content.Client/SubFloor/TrayScannerVisualizer.cs

```diff
@@ -0,0 +1,35 @@
+using Robust.Client.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Maths;
+
+namespace Content.Client.SubFloor;
+
+public class TrayScannerVisualizer : AppearanceVisualizer
```

Comment:
considering this is on the t-ray-shown objects and not the scanner itself, this should probably get a more apt name